boltdesignsystem/bolt
# DS-529: Pega.com: Add 'Download slides' CTA to PW Replays "Best Of" cards

## Jira

https://pegadigitalit.atlassian.net/browse/DS-529

## Summary

Download slides CTA was added. All phases from the `Best-of-content` were removed and only one final version was kept.

## Details

Download slides CTA was added to the `bolt-teaser` component and to the `best-of-content` pages in relation to the proposed wireframe.

## How to test

Pull the branch. Check out the `bolt-teaser` component and `best-of-content` pages where the new `Download slides` CTA was added. Make sure the new CTA is correct in terms of semantics and accessibility.

## Release notes

- Added new CTA to the Listing Teaser actions list on the `bolt-teaser` component and the `best-of-content` page.

### Visual Changes

#### Teaser component

- Action list is now aligned to the left.
